src/main/java/frc/robot/RobotContainer.java

```diff
@@ -106,27 +106,32 @@ public class RobotContainer {
    * ***********************
    */
 
-  private SendableChooser<Command> autoChooser;
-
   /* The CommandXboxController instance must be static to allow the getter methods for its axes
    * to work.
    */
   /** Xbox controller for the driver. */
   public static CommandXboxController driverController =
       new CommandXboxController(Constants.Controllers.DRIVER_CONTROLLER_PORT);
 
+  /** Sendable Chooser for path planning autos. */
+  private final SendableChooser<Command> autoChooser;
+
   /** Constructor for {@link RobotContainer} */
   public RobotContainer() {
     configureBindings();
     configureAuto();
 
     drive.setDefaultCommand(driveRobotOriented);
+
+    /* Build an auto chooser. This will use Commands.none() as the default option. */
+    autoChooser = AutoBuilder.buildAutoChooser();
+    /* Another option that allows you to specify the default auto by its name */
+    /* autoChooser = AutoBuilder.buildAutoChooser("My Default Auto"); */
+    SmartDashboard.putData("Auto Chooser", autoChooser);
```

Comment:
Move this stuff into `configureAuto()`

Reply:
To clarify: All auto configuration should happen in `configureAuto()`. The only stuff that should be outside of `configureAuto()` is the SendableChooser instance var and the getAutonomousCommand() method.